.envrc

```diff
@@ -0,0 +1 @@
+dotenv .env.development
```

Comment:
Should we be including the .envrc? This will cause issues for people that want to use direnv to configure their local environment but don't want to commit that config to the repo. Would it be better to explain what to add to the .envrc in the docs?

Reply:
have linked to using direnv it in the file `docs/run-local-docker.md`

Reply:
.envrc should be ignored by git though and added locally. Maybe make this .envrc.example if you want to have it in the repo?